Documentation/using-rkt-with-systemd.md

```diff
@@ -172,32 +155,32 @@ To make socket activation work, you need to add to your application manifest a [
 }
 ```
 
-Then you will need a pair of service and socket unit files. The socket unit file will have the same port you've set in the image manifest of your app and the service unit file will just run `rkt`:
+Then you will need a pair of `.service` and `.socket` unit files. The socket unit file will have the same port you've set in the image manifest of your app, and the service unit file will run `rkt`:
 
 ```
-# my-socket-activated-app.service
+# my-socket-activated-app.socket
 [Unit]
-Description=My socket-activated app
+Description=My socket-activated app's socket
 
-[Service]
-ExecStart=/usr/bin/rkt run myapp.com/my-socket-activated-app:v1.0
-KillMode=mixed
+[Socket]
+ListenStream=8080
 ```
 
 ```
-# my-socket-activated-app.socket
+# my-socket-activated-app.service
 [Unit]
-Description=My socket-activated app's socket
+Description=My socket-activated app
 
-[Socket]
-ListenStream=8080
+[Service]
+ExecStart=/usr/bin/rkt run myapp.com/my-socket-activated-app:v1.0
+KillMode=mixed
 ```
 
-Finally you start the socket unit:
+Finally, start the socket unit:
 
 ```
-# systemctl start my-socket-activated-app.socket
-# systemctl status my-socket-activated-app.socket
+$ systemctl start my-socket-activated-app.socket
```

Comment:
@robszumski It's definitely common in shell-heavy tutorials or anything like that. However, I'm not totally sold on it, because that non-alphanum. char gets loaded with quite a lot of context. For now, however, in this repo, it's a common practice and I don't want to change it in just one document. So I walked it back in #2078